proposed/2023/pmui-tab-consolidation.md

```diff
@@ -0,0 +1,71 @@
+
+
+
+### Technical explanation
+
+All packages with an update available will have a checkbox next to it, with the option at the top of the screen to select all, and bulk update all packages with an available update, or handle them on a one-at-a-time basis. This behaves similarly to the current functionality of the "Updates" tab
+
+## Drawbacks
+
+The current layout with the three tabs provides an inutitive design with a clear, seperate function for each of the tabs (browse packages, view the packages I have installed, update packages). Moving away from this concept might cause a disruption to users who have been using our platform for a long time, and are familiar with this current experience. However, based on feedback as well as a movement towards a more modern UI, designing a simpler and more consolidated UI will keep NuGet relevant and ultimately adress many painpoints of the newer developer demographic.
+
+Additionally, consolidating all of this informaiton onto one tab would create one busier tab as opposed to two tabs with less elements/clutter. However, we want to utilize and make the most of available "real estate" on your UI, and minimize the amount of "clicking around" that customers have to do to accomplish a task. 
+
+## Rationale and alternatives
+
+Alternatives to be considered int his design could be the placement of specific elements on the screen, as well as the way in which bulk update capabilities can be transferred to this new UI concept. The current proposed design bubbles updates up to the top, but we could also consider the concept of putting it over to the side of the screen, or doing "nested tabs" where within the "My Packages tab, the user would see two tabs -- one with all installed packages and one just with packages with available updates. However, this "nested tab" alternate design could be slightly counterintuitive, as it would achieve the goal of condensing the highest level of segmentation within the UI (the top level tabs), but would likely not do much for reducing the amount of clicks required between tabs to accomplish the task of updating a package. 
+
+
+## Prior Art
+
+The basis of this design comes from other similar and more modern virtual software marketplace designs we see today such as the Microsoft Store app for Windows. The current experience in the Microsoft Store allows for the user to browse apps of different types (Apps, Games, Entertainment), and then there is one single tab view titled "Library" which displays all of the different apps that a user has installed, and when an update is available in one of these apps, it is bubbled up to the top and the user can install the update. Once the update has completed, the app resturns the the list with the rest of the up to date apps. The end-goal experience for NuGet would be similar to this. 
+
+Below shows a screenshot of how this experience looks when an app on the Microsoft Store has an update available.
```

Comment:
Drawing inspiration from this is reasonable. 
To only thing for me is that in the MS store, you nearly *always* want to be on latest, so updates being first makes a lot of sense. With packages that's not always the case as there are more things that come with updating a package (tests/runtime validaiton and sometimes just ignoring certain updates for many different reasons.).